spherogram_src/graphs.py

```diff
@@ -1074,8 +1073,8 @@ def closed_subsets(self):
             if 0 < len(X):
                 pairwise_incomparable = True
                 for x in X:
-                    if ( X.intersection(self.smaller[x]) or
-                         X.intersection(self.larger[x]) ):
+                    if (X.intersection(self.smaller[x]) or
+                            X.intersection(self.larger[x])):
```

Comment:
Perhaps remove one more space? Also PEP8 recommends:
```python
                    if (X.intersection(self.smaller[x])
                           or X.intersection(self.larger[x])):
```
See [Break line before or after binary operator](https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator). But that could be another ticket.